VTKConverter/VTKConverter/Program.cs

```diff
@@ -8,8 +8,8 @@ class Program
 
         static void Main(string[] args)
         {
-            LoggingConfiguration.Configure();
-            Log.Info("Program Started!");
+            var logConfig = new LoggingConfiguration();

```

Comment:
Why you've changed it this way?

Reply:
I've committed review changes to the wrong branch and then cherrypicked it, it might be the reason why it was changed back to that version. I've fixed it and looked through the commit to ensure that's the only mistake.